ste/sender_blockBlob_test.go

```diff
@@ -53,3 +57,43 @@ func TestGetVerifiedChunkParams(t *testing.T) {
 	_, _, err = getVerifiedChunkParams(transferInfo, memLimit, memLimit)
 	a.Equal(expectedErr, err.Error())
 }
+
+func TestDeleteDstBlob(t *testing.T) {
```

Comment:
can you add a test case testing a situation where we shouldnt delete the blob?

Reply:
I could, I would need to make a mock Job Part Transfer Manger as seen in your PR: https://github.com/Azure/azure-storage-azcopy/pull/2430/files#diff-3435557beb691bbe572a99ced6d79bbaf691a0638c9d72f67a48bbeb10f688b0 

Then I can use that to run Prologue instead since the actual method does not check for the flag.

Reply:
Done!